### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by 0xMALVEE**
Page titles - updated the page title
Image alt text - updated all alt text without a value
Text headings - no issues found
Colour contrast - no issues found
Resize - no issues found
Moving content - None in the project
Multimedia - None in the project
Basic structure of page - Already in place

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,7 +78,7 @@ <h4 class="letsconnect">LET'S CONNECT</h4>
     <br />
     <section class="works">
       <div class="work-cards tonic 1">
-        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/tonic1.png" alt="" /></div>
+        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/tonic1.png" alt="Work card image" /></div>
```
Comment (Jeddaa) [inline index.html:81]: Updated the alt text for the image

File: index.html [hunk 2]
```diff
@@ -129,8 +129,8 @@ <h2 id="h-tag">Tonic</h2>
         </div>
       </div>
       <div class="work-cards multipost two">
-        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/multipost1.png" alt="" /></div>
-        <div class="desktop-work-img"><img src="icons/multipost2.png" alt="" /></div>
+        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/multipost1.png" alt="Work card image" /></div>
```
Comment (Jeddaa) [inline index.html:132]: Updated the alt text for the image

File: index.html [hunk 3]
```diff
@@ -237,7 +237,7 @@ <h2 id="h-tag" class="desktop-multipost">Facebook 360</h2>
         </div>
       </div>
       <div class="work-cards uber two">
-        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/multipost2.png" alt="" /></div>
+        <div class="mobile-work-img"><img class="mobile-work-img" src="icons/multipost2.png" alt="Work card image" /></div>
```
Comment (Jeddaa) [inline index.html:240]: Updated the alt text for the image

File: index.html [hunk 4]
```diff
@@ -326,15 +326,15 @@ <h3 class="lang-header">Languages</h3>
             <div class="lang">
               <ul class="lang-ul">
                 <li>
-                  <img class="lang-img" src="./icons/javascript.png" alt="" />
```
Comment (Jeddaa) [inline index.html:329]: Updated the alt text for the image

File: index.html [hunk 5]
```diff
@@ -326,15 +326,15 @@ <h3 class="lang-header">Languages</h3>
...
+                  <img class="lang-img" src="./icons/javascript.png" alt="Javascript Icon" />
                   <p>Javascript</p>
                 </li>
                 <li>
-                  <img class="lang-img" src="./icons/html.png" alt="" />
```
Comment (Jeddaa) [inline index.html:333]: Updated the alt text for the image

File: index.html [hunk 6]
```diff
@@ -326,15 +326,15 @@ <h3 class="lang-header">Languages</h3>
...
+                  <img class="lang-img" src="./icons/html.png" alt="Html Icon" />
                   <p>HTML</p>
                 </li>
                 <li>
-                  <img class="lang-img" src="./icons/css.png" alt="" />
```
Comment (Jeddaa) [inline index.html:337]: Updated the alt text for the image

**Review by Jeddaa — COMMENTED** [review]
Page titles - updated the page title
Image alt text - updated all alt text without a value

**Review by shella12 — APPROVED** [review]
Hi @0xMALVEE , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Report includes all points ✔️ 
- Updated code for accessibility 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**